PF/Assignment1/Sort/Sort.java

```diff
@@ -0,0 +1,44 @@
+//main class
+public class Sort {
+
+	// main function
+	public static void main(String arg[]) {
+		MenuAndPrint menuandprint = new MenuAndPrint();
+		int[] array = menuandprint.arrayInput();
+		int length = array.length;
```

Comment:
ACTION: no need to create a new variable to store length of array use directly array.length.